command/commands/user.py

```diff
@@ -113,3 +119,50 @@ def handle(self, command, user_id):
             if args.bio is not None:
                 msg += "bio: {}".format(args.bio)
             return msg
+
+    def delete_helper(self, user_id, slack_id, channel):
+        """
+        Delete user from database.
+
+        Delete user with slack_id from database if user with user_id has
+        admin permission level.
+
+        :param user_id: Slack ID of user who is calling the command
+        :param slack_id: Slack ID of user who is being deleted
+        :param channel: ID of Slack channel that called command
+        :return: returns permission error message if not admin,
+                 returns deletion message if user is deleted.
+        """
+        message = "Deleted user with Slack ID: " + slack_id
```

Comment:
I’m not sure it’s worth having `message` as a local variable, seeing as how it’s only used in one place (though I’d be happy to hear arguments for/against).

Reply:
Since these are constants, they should be in the static scope. Something like `self.delete_text` or `self.delete_status`.

Reply:
We may eventually want to read them from a config file as well - storing static strings in config is quite common in mobile development. That should definitely be a future PR though.